### PR TITLE
parser: Create SECURITY.md

### PR DESCRIPTION
**Pull request by s3nt3**
### What problem does this PR solve? 

Announce the way and specifications for users to submit vulnerabilities to the tidb security team through Github-Security-Policy.

### What is changed and how it works?

Refer to the steps mentioned in [add a security policy to your repository](https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository) to create SECURITY.md.

### Check List

Tests
 
 - No code

### CODE REVIEW

**Comment by tangenta** [comment]
> TiDB is a fast-growing open source database.
> The TiDB security team will confirm the vulnerabilities and contact you within 2 working days after your submission.

I don't think these sentences are appropriate because TiDB is not the only user using this repo.

**Comment by s3nt3** [comment]
> > TiDB is a fast-growing open source database.
> > The TiDB security team will confirm the vulnerabilities and contact you within 2 working days after your submission.
> 
> I don't think these sentences are appropriate because TiDB is not the only user using this repo.

**Comment by s3nt3** [comment]
Hi, @tangenta the main purpose of adding this security policy is to inform users how to submit vulnerabilities. As the sql parser of tidb, the tidb security team is very concerned about the security issues of this repo. We hope that we can accept the security issues reported by the community, and we will also invest in support. Based on the above reasons, I have synchronized tidb's security policy. If the reviewer and maintainer think that the above statement is inappropriate, welcome to put forward specific suggestions for modification, and I will modify the policy document.

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Security Vulnerability Disclosure and Response Process
+
+TiDB is a fast-growing open source database. To ensure its security, a security vulnerability disclosure and response process is adopted.
```
Comment (tangenta) [inline SECURITY.md:3]:
```suggestion
```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+The primary goal of this process is to reduce the total exposure time of users to publicly known vulnerabilities. To quickly fix vulnerabilities of TiDB products, the security team is responsible for the entire vulnerability management process, including internal communication and external disclosure.
```
Comment (tangenta) [inline SECURITY.md:3]:
```suggestion
The primary goal of this process is to reduce the total exposure time of users to publicly known vulnerabilities. TiDB security team is responsible for the entire vulnerability management process, including internal communication and external disclosure.
```

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Security Vulnerability Disclosure and Response Process
+
+The primary goal of this process is to reduce the total exposure time of users to publicly known vulnerabilities. To quickly fix vulnerabilities of TiDB products, the security team is responsible for the entire vulnerability management process, including internal communication and external disclosure.
+
+If you find a vulnerability or encounter a security incident involving vulnerabilities of TiDB products, please report it as soon as possible to the TiDB security team (security@tidb.io).
```
Comment (tangenta) [inline SECURITY.md:5]:
```suggestion
If you find a vulnerability or encounter a security incident involving vulnerabilities of this repository, please report it as soon as possible to the TiDB security team (security@tidb.io).
```

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+The asterisk (*) indicates the required field.
+
+# Response Time
+
+The TiDB security team will confirm the vulnerabilities and contact you within 2 working days after your submission.
+
+We will publicly thank you after fixing the security vulnerability. To avoid negative impact, please keep the vulnerability confidential until we fix it. We would appreciate it if you could obey the following code of conduct:
+
+The vulnerability will not be disclosed until TiDB releases a patch for it.
```
Comment (tangenta) [inline SECURITY.md:29]:
```suggestion
The vulnerability will not be disclosed until a patch is released for it.
```

**Comment by tangenta** [comment]
Rest LGTM

**Comment by s3nt3** [comment]
@tangenta Thanks a lot for your suggestions, I have updated it.

**Review by tangenta — APPROVED** [review]

**Review by dveeden — APPROVED** [review]

**Review by morgo — APPROVED** [review]

**Pull request merged**